CLAUDE.md

```
@@ -39,98 +39,80 @@ This is a monorepo with three main modules:
-npm run dev      # Development server
-npm run build    # Production build
-npm run lint     # Linting
-npm run test     # Run tests
+npm install            # Install dependencies
+npm run dev            # Development server (with Turbopack)
+npm run build          # Production build
+npm run start          # Start production server
+npm run lint           # ESLint checking
 ```
 
 **Infrastructure**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)